java/semantickernel-core/src/main/java/com/microsoft/semantickernel/templateengine/blocks/TextBlock.java

```diff
@@ -25,18 +23,4 @@ public String render(ContextVariables variables) {
     public TextBlock(String text, int startIndex, int stopIndex) {
         super(text.substring(startIndex, stopIndex), BlockTypes.Text);
     }
-    /*
```

Comment:
Why is this implementation not being included?

Reply:
This class does not offer any functionality over the super class